packages/uicore/src/components/Button/Button.css

```diff
@@ -131,6 +131,20 @@
 
       &.variation-secondary {
         --focus-outline-color: var(--primary-7);
+        &[class*='intent-success'] {
+          border: 1px solid var(--green-800) !important;
```

Comment:
we can avoid `!important` but base style comes with it and removing it there could casue lot of diff in NgUI